src/utils/tableUtils/getRequiredDataFields.ts

```diff
@@ -11,5 +11,5 @@ export function getRequiredDataFields<ColumnId extends string, RequiredField ext
         return fields;
     }, new Set<RequiredField>());
 
-    return Array.from(requiredFieldsSet);
+    return Array.from(requiredFieldsSet).sort();
```

Comment:
Sort to ensure no additional requests when order changes.

Example:
Columns `[DiskSpace, Read]`, data fields `[State, Read]`
Columns `[Read, State]`, data fields `[Read, State]`

Although it's the same data, it will be requested twice, because of different state keys (`"fieldsRequired:State,Read"` vs `"fieldsRequired:Read,State"`